### ISSUE TITLE
[Backend] Documentation for initial backend setup

### ISSUE THREAD

**Issue by jackfrost13** [HITK-TECH-Community/Community-Website#82]
### Description

Prepare a documentation for the initial backend setup.
Checkout `backend-setup` branch, go through the entire codebase and prepare a well descriptive documentation of it which will be used in the wiki page for the project. 

This will help you in getting your basics cleared and learn a lot about backend setup and development(our experiences) and will be a unique experience where you get to interact with the mentors on 1-1 interaction (discussions). We would love to share our experiences and exchange knowledge with you.

This will call for several revisions and discussions for which feel free to contact us via slack.

**Multiple people can be assigned this issue as this can be a group work**

**Comment by tanay13** [comment]
Can i work on this issue??

**Comment by jackfrost13** [comment]
Sure, go ahead. Go through the code once and then we can discuss more on it

**Comment by tanay13** [comment]
@jackfrost13 Yeah sure !!

**Comment by tanay13** [comment]
@jackfrost13 @Kajol-Kumari Please provide me with some backend documentation so that i get an idea how the documentation should be

**Comment by jackfrost13** [comment]
Surely, DM me on slack!

**Comment by jackfrost13** [comment]
Have assigned you one of the requested issues, please complete that first

**Comment by jackfrost13** [comment]
let us know on which module you will be working

**Comment by manishaag7** [comment]
Sure @jackfrost13

**Comment by jackfrost13** [comment]
@manishaag7 Please let us know on which module you are working?

**Comment by manishaag7** [comment]
I will tell you by tonight, sorry for the delay

**Comment by manishaag7** [comment]
Can I work on url shortener module, can you also provide some guide or example how to write backend documentation? 
Thank you!

**Comment by Ashuto7h** [comment]
Why there is a need for this issue? 
The backend setup guide is already available at Readme.

**Comment by jackfrost13** [comment]
@Ashuto7h Yes the initial guide has been prepared but it doesn't contain all the documentation required.
This issue serves as an umbrella under which multiple issues(module specific is created)

**Comment by Gurdeep475** [comment]
I would like to work on this issue under GSSOC`22. If no one is working on it. @jackfrost13
